pkg/clients/aws/client.go

```diff
@@ -92,23 +94,38 @@ func (c *client) getAccountNumber(ctx context.Context) (string, error) {
 }
 
 var (
-	productSKUField                = "ProductSKU"
-	rancherProductSKUNonEmea       = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
-	rancherProductSKUEmea          = "a303097d-1dc2-4548-8ea6-f46bb9842e21"
-	maxResults               int32 = 1
+	productSKUField          = "ProductSKU"
+	rancherProductSKUNonEmea = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
+	rancherProductSKUEmea    = "a303097d-1dc2-4548-8ea6-f46bb9842e21"
+	// test skus - should not be enabled at the same time as prod skus
+	rancherProductTestSKUNonEmea       = "83929a73-7c49-4511-aa45-8854a4f001d4"
+	rancherProductTestSKUEmea          = "e001cf36-9e45-496e-be2c-b48749bf7dd2"
+	maxResults                   int32 = 1
 )
 
 func (c *client) GetRancherLicense(ctx context.Context) (*types.GrantedLicense, error) {
-	license, err := c.getLicenseForProductID(ctx, rancherProductSKUNonEmea)
-	if err != nil {
-		// if we could not get the original license, attempt to retrieve the license for Emea countries
-		license, newErr := c.getLicenseForProductID(ctx, rancherProductSKUEmea)
-		if newErr != nil {
-			return nil, fmt.Errorf("unable to get license for non-emea: %s, unable to get license for emea: %s", err.Error(), newErr.Error())
+	// only attempt to retrieve the Emea licenses if we can't get the standard license
+	productSKUs := []string{rancherProductSKUNonEmea, rancherProductSKUEmea}
+	// test product IDs should only be used specifically when requested
+	if c.useTestProducts {
+		productSKUs = []string{rancherProductTestSKUNonEmea, rancherProductTestSKUEmea}
+	}
+	var errors []error
+	for _, productSKU := range productSKUs {
+		license, err := c.getLicenseForProductID(ctx, productSKU)
+		if err != nil {
+			errors = append(errors, fmt.Errorf("unable to get license for sku %s: %w", productSKU, err))
+			continue
 		}
+		// if we found a valid license, return the first one that we find
 		return license, nil
 	}
-	return license, nil
+	// if we got to this point, then we never found a valid license
+	err := fmt.Errorf("unable to get a valid rancher license")
+	for _, productError := range errors {
```

Comment:
I'm not sure what you're trying to do here. Is the goal to have a single error with for a single product? If you have multiple values in `errors`, then you'd overwrite the original message `unable to get a valid rancher license` in the first error.

Reply:
The goal is to aggregate the errors into one error message. I'd like to be able to see the various errors for each potential product ID - it's possible that one of the product skus was valid for this account and we just weren't able to get it for whatever reason.

> If you have multiple values in `errors`, then you'd overwrite the original message `unable to get a valid rancher license` in the first error.

I don't think you are correct here. In a simple `main.go` file, this code:
```golang
package main

import (
	"fmt"
)

func main() {
	error1 := fmt.Errorf("error1")
	error2 := fmt.Errorf("error2")
	err := fmt.Errorf("initial error")
	errors := []error{error1, error2}
	for _, productError := range errors {
		err = fmt.Errorf("%s, %s", err.Error(), productError.Error())
	}
	fmt.Printf("%v \n", err)
}

```
produces the output: `initial error, error1, error2`, which combines the errors as expected. If you see a significant difference between the sample code that I've provided and the code on the PR (or see a place where you disagree), please let me know.

Reply:
It might be worth clarifying in a comment on this what I'm trying to do, as I can see how it would be confusing. If I push any other changes I'll try to remember to make a comment here as well.